### DISCUSSION TITLE
文章一级目录跳转到文章首行

### DISCUSSION THREAD

**Discussion by ClimberZheng** [ppoffice/hexo-theme-icarus#902] [Q&A] [answered]
配置完成之后，所有一级目录点击并不能实现跳转到该位置，而是都跳转到了首行，鼠标放在一级目录的时候，浏览器显示的地址是文章的所在地址，二级目录和三级是正常的，

在主题的博客当中，一级标题是可以跳转的，请问是我哪里配置遗漏了吗还是有什么其它问题，能否麻烦大神给点修改的建议呢

![image](https://user-images.githubusercontent.com/44958459/127313979-32ba3c58-619d-44b0-a903-78eee25a80c0.png)

**Comment by ppoffice** [comment, marked as answer]
你是不是安装了什么与目录有关的hexo插件？

**Reply by ClimberZheng** [reply]
好的，我去排查下，感谢您的建议
